ansible/roles/common/tasks/main.yml

```diff
@@ -23,3 +23,40 @@
   apt: name="libssl1.0.0" state=present
   tags:
     - upgrade_openssl
+
+
+- name: Hard file limits
+  sudo: yes
+  pam_limits:
```

Comment:
This module is in ansible `>2.0`, so this is failing on our prod ansible (`1.9.6`). Are we planning to upgrade our ansible?